js_tutorial/exercises/004_palindrome.js

```diff
@@ -0,0 +1,34 @@
+//4. Enter a  string and find if it's a palindrome
+
+const string = "malayalam";
+const palindrome = (str) => {
+    let reverse = str.split("").reverse().join("");   //using join("") to eliminate commas while joining the string
```

Comment:
Try implementing the split() reverse() join()  built in functions from the scratch.

Reply:
Will do!